compilers/concrete-compiler/compiler/include/concretelang/Support/V0Parameters.h

```diff
@@ -80,6 +80,10 @@ const concrete_optimizer::Encoding DEFAULT_ENCODING =
 const bool DEFAULT_CACHE_ON_DISK = true;
 const uint32_t DEFAULT_CIPHERTEXT_MODULUS_LOG = 64;
 const uint32_t DEFAULT_FFT_PRECISION = 53;
+const uint64_t DEFAULT_LOG2_POLYNOMIAL_SIZE_MIN = std::numeric_limits<uint64_t>::min();
+const uint64_t DEFAULT_LOG2_POLYNOMIAL_SIZE_MAX = std::numeric_limits<uint64_t>::max();
+const uint64_t DEFAULT_GLWE_DIMENSION_MIN = std::numeric_limits<uint64_t>::min();
+const uint64_t DEFAULT_GLWE_DIMENSION_MAX = std::numeric_limits<uint64_t>::max();
```

Comment:
This encodes a kind of soft `Optional` when it arrives in `default_cpu` in `config.rs`. I think an explicit optional would be better here.